src/astra_driver.cpp

```diff
@@ -278,7 +280,10 @@ void AstraDriver::configCb(Config &config, uint32_t level)
 
   use_device_time_ = config.use_device_time;
 
-  data_skip_ = config.data_skip+1;
+  // Todo: add seperate data skip for all streaming modes
+  ir_data_skip_ = config.ir_data_skip+1;
```

Comment:
Is TODO done here?

Reply:
That's already done. Good catch, I'll remove the comment.